src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -138,3 +139,10 @@ source_metadata:
       object_source: http://purl.obolibrary.org/obo/doid.owl
       object_source_version: http://purl.obolibrary.org/obo/doid/releases/2018-07-05/doid.owl
       mapping_date: "2018-07-05"
+  - from: MONDO
+    to: GARD
```

Comment:
Ah yes, I figured this probably needed to be added, w/ the way this is currently set up.

Reply:
I added this as a placeholder for now. But the true trigger for GARD being mapped is a few lines above this one under `subject_prefixes:` (line 89)